buildSrc/src/main/java/io/opentelemetry/instrumentation/gradle/AutoInstrumentationPlugin.java

```diff
@@ -43,6 +51,40 @@ public void apply(Project project) {
             });
   }
 
+  private void addDependencies(Project project) {
```

Comment:
Am I right that all projects that use auto-instrumenation-plugin will add the following dependencies?

Reply:
Correct!